### DISCUSSION TITLE
http request con ricerca codice fiscale

### DISCUSSION THREAD

**Discussion by durandal2** [fattureincloud/api#375] [Q&A] [answered]
ciao,
ho necessità di interrogare il database con una richiesta http se un codice fiscale risulti già presente nel database clienti
ho consultato il documento di supporto https://developers.fattureincloud.it/api-reference#get-/c/-company_id-/entities/clients/-client_id-

ma non ho trovato alcuna possibilità al riguardo.
come posso fare?
grazie

**Comment by valmoz** [comment, marked as answer]
Salve @durandal2 
può usare i [Filtri](https://developers.fattureincloud.it/docs/basics/additional-capabilities/filter-results) offerti dalle nostre API.

Mauro
